### ISSUE TITLE
Request for Updating the AUR Package with the Latest Binary Release

### ISSUE THREAD

**Issue by theSprog** [out-of-cheese-error/the-way#161]
Hello, the-way team

I am an user of `the-way` and I greatly appreciate the work you've put into developing and maintaining this tool. It has significantly improved my workflow, and I believe many others in the Arch Linux community feel the same.

I noticed that the AUR package for `the-way` ([[the-way-git](https://aur.archlinux.org/packages/the-way-git/)](https://aur.archlinux.org/packages/the-way-git/)) is currently pointing to an older version of the software. The last commit to the AUR package was made about 2 years ago, which means Arch Linux users are not benefiting from the latest features and improvements you've implemented since then.

Given the recent updates and enhancements you've made to `the-way`, I kindly request if you could update the AUR package, or possibly create a new AUR package for the latest binary release. This would make it much easier for Arch Linux users to install and enjoy the latest version of `the-way`.

Thank you for considering this request, and thank you again for your contributions to the open-source community through your work on `the-way`.

**Comment by Ninjani** [comment]
Great to hear that you find it useful! 

The AUR package was meant to be [up to date with the latest tagged commit on master](https://github.com/out-of-cheese-error/the-way/pull/81#issuecomment-711252228) from what I understood.

@spikecodes is there something to change to make this up to date again?

**Comment by spikecodes** [comment]
Hey! I found out my script to update the versions automatically doesn't actually work because the AUR [requires an external script](https://www.reddit.com/r/archlinux/comments/z3gjz0/how_does_aur_page_update_to_actual_version_of_git/) to update the PKGBUILD files for each package. I unfortunately don't have the time to maintain the AUR packages that I used to but I unlisted myself from the package maintainer list so that others can now adopt them.

If you login to the AUR and visit https://aur.archlinux.org/packages/the-way-git, you'll see the "Adopt Package" option on the right sidebar which will let you set the `pkgver` variable (in the PKGBUILD) to the latest version number from GitHub Releases.

This is the only change needed to keep it synced to GitHub. Alternatively, you could setup a simple CI script to automate this from GitHub Actions.

**Comment by theSprog** [comment]
> Hey! I found out my script to update the versions automatically doesn't actually work because the AUR [requires an external script](https://www.reddit.com/r/archlinux/comments/z3gjz0/how_does_aur_page_update_to_actual_version_of_git/) to update the PKGBUILD files for each package. I unfortunately don't have the time to maintain the AUR packages that I used to but I unlisted myself from the package maintainer list so that others can now adopt them.
> 
> If you login to the AUR and visit https://aur.archlinux.org/packages/the-way-git, you'll see the "Adopt Package" option on the right sidebar which will let you set the `pkgver` variable (in the PKGBUILD) to the latest version number from GitHub Releases.
> 
> This is the only change needed to keep it synced to GitHub. Alternatively, you could setup a simple CI script to automate this from GitHub Actions.

sure, I download the Git Clone URL and modified the pkgver to 0.20.3, and the I makepkg，but I got this error：

          /usr/sbin/ld: /mnt/c/Users/fan/Desktop/the-way-git/src/the-way/target/release/deps/the_way-be0228cc1a6a0fe5: hidden symbol `ring_core_0_17_7_OPENSSL_ia32cap_P' isn't defined
          /usr/sbin/ld: final link failed: bad value
          collect2: error: ld returned 1 exit status

  = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib)

But when I downloaded the source code and compiled it，it built fine!
I am an aur rookie so maybe I do something wrong along the way😥

**Comment by spikecodes** [comment]
Hmm, not sure for that one. Maybe look for a Rust-specific guide to packaging for the AUR? Or perhaps ask ChatGPT/Claude.

**Comment by theSprog** [comment]
> Hmm, not sure for that one. Maybe look for a Rust-specific guide to packaging for the AUR? Or perhaps ask ChatGPT/Claude.

Indeed, it is weird. When I makepkg -si directly I get this error, but when I cd src/the-way and then cargo build --locked --release everything works fine. build() function is written like this, I didn't change it and it looks working fine:
build() {
	cd "${_pkgname}"
	cargo build --locked --release
}

I'm not sure what's wrong with this, or if it's a bug in makepkg ?
